doc/usecases/mount/UC_edit_rdb.md

```diff
@@ -0,0 +1,49 @@
+# Use Case: Edit a configuration value stored in a relational database
+
+## Summary
+
+- **Scope:** Mounting
+- **Level:** Sea
+- **Actors:** Administrators
+- **Brief:** The administrators can use Elektra to change key-value based configuration data stored in a supported relational database.
+- **Status:** Draft
+
+## Scenarios
+
+- **Precondition:**
+
+  - The administrators have a working installation of Elektra and a tool to access the KDB (key database) on their system.
+  - The database is mounted into the KDB. (see [Use Case: Mount a data source](./UC_mount.md))
+
+- **Main success scenario:**
+
+  - The administrators query a value from the mounted database by giving the unique keyname. (see [Use Case: Read a configuration value from a file](./UC_read_rdb.md))
+  - Elektra reads the current (old) value from the mounted database and returns it to the administrators.
+  - The administrators use Elektra to change the value of the given key, by giving the unique keyname and the new value.
+  - Elektra updates the affected tuple in the database.
+  - The administrators query the same key as in the first step.
+  - Elektra returns the new value of the given key to the administrators.
+  - The administrators query the database directly to verify that the requested change was written.
+
+- **Alternative scenario:**
+
+  - The requested key is not found in the database.
+    - Elektra reports the error to the administrators.
```

Comment:
As Elektra simply `set` whatever is given, the non-presence of keys shouldn't matter (they get created/deleted as demanded).

Reply:
Thank you for the remark! I changed the scenario.